File: En Ruby - Interpretado/CombinarListas.rb
#!/usr/bin/env ruby

listUno = ["Negro", "Blanco", "Verde", "Cafe", "Gris"]
listDos = ["Grande", "Mediano", "Chico"]

listUno.each { |l1| listDos.each { |l2| puts "#{l1}-#{l2}" } }

File: En PHP - Interpretado/CombinarListas.php
<?php
$listUno = array("Negro", "Blanco", "Verde", "Cafe", "Gris");
$listDos = array("Grande", "Mediano", "Chico");

for($m = 0; $m < count($listUno); $m++) {
  for($n = 0; $n < count($listDos); $n++) {
    echo "{$listUno[$m]}-{$listDos[$n]}\n";
  }
}
?>
